src/components/SkipLink/SkipLink.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import SkipLink from './SkipLink';
+
+const meta = {
+  title: 'Components/SkipLink',
+  component: SkipLink,
+  argTypes: {
+    destination: {
+      description: 'Human-readable name for destination appended to skip link text',
+      controlType: 'text'
+    },
+    destinationId: {
+      description: `Value of destination's id attribute`,
```

Comment:
Can we add the descriptions as comments in the component props (see accordion)?
That way the descriptions are present in code too